extensions/guacamole-vault/modules/guacamole-vault-hv/src/main/resources/translations/en.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This file also needs a `DATA_SOURCE_HASHCORP_VAULT` section with a `NAME` entry. See https://github.com/apache/guacamole-client/blob/main/extensions/guacamole-vault/modules/guacamole-vault-ksm/src/main/resources/translations/en.json for the matching example for the KSM module.